util/bklog/log.go

```diff
@@ -0,0 +1,46 @@
+package bklog
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel/trace"
+)
+
+var (
+	G = GetLogger
+	L = logrus.NewEntry(logrus.StandardLogger())
+)
+
+type (
+	loggerKey struct{}
+)
+
+// WithLogger returns a new context with the provided logger. Use in
+// combination with logger.WithField(s) for great effect.
+func WithLogger(ctx context.Context, logger *logrus.Entry) context.Context {
```

Comment:
Don't see this used anywhere. Maybe not add it until there is a need.

Reply:
Used here, may with section id for logs. 
so we could know which logs belongs to sections 

https://github.com/moby/buildkit/pull/2236/files#diff-527db53529b974ea055e9d1d51a63c3c0b7dbf8cc61b03eb3d9693604333a437R112